ui-v2/app/templates/settings.hbs

```diff
@@ -16,25 +16,34 @@
         <fieldset>
           <h2>Dashboard Links</h2>
           <p>
-          Add a link to the service detail page in the UI to get quick access to a service-wide metrics dashboard. Enter the dashboard URL into the field below. You can use the placeholders <code>{{'{{Datacenter}}'}}</code> and <code>{{'{{Service.Name}}'}}</code> which will be replaced with the name of the datacenter/service currently being viewed.
+            Add a link to the service detail page in the UI to get quick access to a service-wide metrics dashboard. Enter the
+            dashboard URL into the field below. You can use the placeholders <code>{{'{{Datacenter}}'}}</code> and
+            <code>{{'{{Service.Name}}'}}</code> which will be replaced with the name of the datacenter/service currently being
+            viewed.
           </p>
           <label class={{concat (if confirming 'confirming') ' type-text'}} id="urls_service">
             <span>Link template for services</span>
-            <input type="text" name="urls[service]" value={{item.urls.service}} onchange={{action 'change'}} onkeypress={{action 'key'}} onkeydown={{action 'key'}} />
-            <em>e.g. https://grafana.example.com/d/1/consul-service-mesh&amp;orgid=1&amp;datacenter={{ '{{Datacenter}}' }}&amp;service-name={{ '{{Service.Name}}' }}</em>
+            <input type="text" name="urls[service]" value={{item.urls.service}} onchange={{action 'change'}}
+              onkeypress={{action 'key'}} onkeydown={{action 'key'}} />
+            <em>e.g. https://grafana.example.com/d/1/consul-service-mesh&amp;orgid=1&amp;datacenter={{ '{{Datacenter}}'
+              }}&amp;service-name={{ '{{Service.Name}}' }}</em>
```

Comment:
Not a biggie for this, but I think your text editor might be auto-wrapping things, let's chat later and take a look.